File: EASY/PASCAL'S TRIANGLE 2/pascalTriangle2.java
class pascalTriangle2 {
    public List<Integer> getRow(int rowIndex){
        List<Integer> row = new ArrayList<>();
        row.add(1);
        for (int i = 1; i <= rowIndex; i++) {
            row.add((int) ((long) row.get(i - 1) * (rowIndex - i + 1) / i));
        }
        return row;
    }    
    public static void main(String[] args) {
        pascalTriangle2 ob = new pascalTriangle2();
        System.out.println(ob.getRow(3));
    }
}

File: EASY/PASCAL'S TRIANGLE 2/pascaltriangle2.c
#include <stdio.h>
void getRow(int rowIndex){
    int arr[rowIndex + 1];
    for (int i = 0; i <= rowIndex; i++){
        arr[i] = 1;
        for (int j = i - 1; j > 0; j--)
            arr[j] += arr[j - 1];
    }
    for (int i = 0; i <= rowIndex; i++)
        printf("%d ", arr[i]);
}
int main(){
    int n = 5;
    getRow(n);
    return 0;
}